.github/workflows/push.yml

```
@@ -100,123 +137,212 @@ jobs:
+
+  publish-release:
+    needs:
+    - prepare
+    - validate
+    - build-standalone-assets
+    - build-legacy-bridge
+    if: ${{ always() && needs.prepare.result == 'success' && needs.validate.result == 'success' && needs.build-standalone-assets.result == 'success' && (needs.build-legacy-bridge.result == 'success' || needs.build-legacy-bridge.result == 'skipped') }}
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/setup-python@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)